cmd/di.go

```diff
@@ -248,8 +248,10 @@ func (di *Dependencies) bootstrapLocationComponents(options node.OptionsLocation
 
 	switch {
 	case options.Country != "":
-		di.LocationResolver = location.NewResolverFake(options.Country)
+		di.LocationResolver = location.NewStaticResolver(options.Country)
+	case options.BuiltInDb:
+		di.LocationResolver = location.NewBuiltInResolver()
```

Comment:
`NewBuiltInResolver` should be default resolver if no other option other resolver given

Reply:
Done